src/tiered/tiered_opener.c

```diff
@@ -19,45 +19,48 @@ __tiered_opener_open(WT_BLOCK_FILE_OPENER *opener, WT_SESSION_IMPL *session, uin
     WT_BUCKET_STORAGE *bstorage;
     WT_DECL_RET;
     WT_TIERED *tiered;
+    u_int open_flags;
     const char *object_name, *object_uri;
+    bool local_only;
 
     tiered = opener->cookie;
     object_uri = NULL;
+    open_flags = flags;
```

Comment:
Do you `open_flags`?  It looks like its use parallels `flags` pretty closely.

Reply:
Good point. Fixed.